roles/common/files/uug_ansible_wrapper.py

```diff
@@ -38,15 +39,15 @@
 USER_CONFIG_PATH = os.path.join(os.environ['HOME'], ".config", "vm_config")
 USER_CONFIG = {
     'git_branch': None,
-    'git_url': "https://github.com/jmunixusers/cs-vm-build",
+    'git_url': DEFAULT_GIT_REMOTE,
     # All roles the user has ever chosen
     'roles_all_time': ["common"],
     # Roles to be used for this particular run
     'roles_this_run': ["common"],
 }
 NAME = "JMU CS VM Configuration"
 VERSION = "Spring 2019"
-
+DEFAULT_GIT_REMOTE = "https://github.com/jmunixusers/cs-vm-build"
```

Comment:
This needs to be moved further up. Notice that it is used in Line 42 but not defined until Line 50? Can you move this either above the comment on line 26 or to line 40 before the  `USER_CONFIG = {`? As it is, the script is unable to run.